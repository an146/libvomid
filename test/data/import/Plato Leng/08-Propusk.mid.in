../../../../test/mid/Plato Leng/08-Propusk.mid