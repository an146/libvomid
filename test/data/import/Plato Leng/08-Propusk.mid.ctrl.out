Tempo:127690 504202 127701 508475 127712 512821 127722 517241 127733 521739 127744 526316 127754 530973 127765 535714 127776 540541 127786 545455 127797 550459 127808 555556 127818 560748 127829 566038 127840 571429 127850 576923 127861 582524 127872 588235 127882 594059 127893 600000 127904 606061 127914 612245 127925 618557 127936 625000 127946 631579 127957 638298 127968 645161 127978 652174 127989 659341 128000 666667 128010 674157 128021 681818 128032 689655 128042 697674 128053 705882 128064 714286 128074 722892 128085 731707 128096 740741 128106 750000 128117 759494 128128 769231 128138 779221 128149 789474 128160 800000 128170 810811 128181 821918 128192 833333 128202 845070 128213 857143 128224 869565 128234 882353 128245 895522 128256 909091 128266 923077 128277 937500 128288 952381 128298 967742 128309 983607 128320 1000000 128330 1016949 128341 1034483 128352 1052632 128362 1071429 128373 1090909 128384 1111111 128394 1132075 128405 1153846 128416 1176471 128426 1200000 128437 1224490 128448 1250000 128458 1276596 128469 1304348 128480 1333333 128490 1363636 128501 1395349 128512 1428571 128522 1463415 128533 1500000 128544 1538462 128554 1578947 128565 1621622 128576 1666667 128586 1714286 128597 1764706 128608 1818182 128618 1875000 128629 1935484 128640 2000000
Time Signature:
Track 0: Name:
Track 0: Notesystem:-1
Track 0: Volume:124
Track 0: Pan:64
Track 0: Balance:
Track 0: Expression:
Track 1: Name:
Track 1: Notesystem:-1
Track 1: Volume:86
Track 1: Pan:64
Track 1: Balance:
Track 1: Expression:
Track 2: Name:
Track 2: Notesystem:-1
Track 2: Volume:106
Track 2: Pan:64
Track 2: Balance:
Track 2: Expression:
Track 3: Name:
Track 3: Notesystem:-1
Track 3: Volume:99
Track 3: Pan:64
Track 3: Balance:
Track 3: Expression:
Track 4: Name:
Track 4: Notesystem:-1
Track 4: Volume:105
Track 4: Pan:64
Track 4: Balance:
Track 4: Expression:
Track 5: Name:
Track 5: Notesystem:0
Track 5: Volume:110
Track 5: Pan:64
Track 5: Balance:
Track 5: Expression:
Channel 0: Program:0 35
Channel 0: Pitch Wheel:
Channel 1: Program:0 25
Channel 1: Pitch Wheel:
Channel 2: Program:0 30
Channel 2: Pitch Wheel:
Channel 3: Program:0 30
Channel 3: Pitch Wheel:
Channel 4: Program:0 42
Channel 4: Pitch Wheel:
Channel 5: Program:
Channel 5: Pitch Wheel:
Channel 6: Program:
Channel 6: Pitch Wheel:
Channel 7: Program:
Channel 7: Pitch Wheel:
Channel 8: Program:
Channel 8: Pitch Wheel:
Channel 9: Program:
Channel 9: Pitch Wheel:
Channel 10: Program:
Channel 10: Pitch Wheel:
Channel 11: Program:
Channel 11: Pitch Wheel:
Channel 12: Program:
Channel 12: Pitch Wheel:
Channel 13: Program:
Channel 13: Pitch Wheel:
Channel 14: Program:
Channel 14: Pitch Wheel:
Channel 15: Program:
Channel 15: Pitch Wheel:
